docassemble/MAVirtualCourt/data/questions/test.yml

```diff
@@ -1,54 +1,6 @@
-metadata:
-  title: Default playground interview
-  short title: Test
-  comment: This is a learning tool.  Feel free to write over it.
```

Comment:
This file was sort of gutted. Is this change needed for this PR?

Reply:
It shouldn't be in the repo--it's a scratch pad. But it's not relied on by anything else.